src/tir/transforms/lower_sparse_buffer.cc

```diff
@@ -354,6 +354,76 @@ class BufferTransformer : public StmtExprMutator {
   bool is_horizontal_fuse_;
 };
 
+class Sparse_indices_buffer_post_process : public StmtExprMutator {
+  public:
+    explicit Sparse_indices_buffer_post_process(){}
+  private:
+    Stmt VisitStmt_(const BlockNode* op) final{
+      if(support::StartsWith(op->name_hint,"binary_search_block"))
+        skip=true;
+      auto ret = StmtExprMutator::VisitStmt_(op);
+      skip=false;
+      return ret;
+    }
+
+    PrimExpr VisitExpr_(const BufferLoadNode* op) final{
+      if(!skip&&support::StartsWith(op->buffer->name,"mid_")&&buffer_processed.count(op->buffer)==0)
+      {
+        buffer_need_process.push_back(GetRef<BufferLoad>(op));
+        find++;
+        return PrimExpr(0);
+      }
+      else
+      {
+        auto find_backup=find;
+        for(auto i : op->indices)
+        {
+          VisitExpr(i);
+        }
+        if(find!=find_backup)
+        {
+          find=find_backup;
+          if(op->buffer->dtype.is_float())
```

Comment:
The data types are not only `float32` and `int32`.
`0` is also not only the valid choice as initial values, suppose we want to perform max reduction, `-inf` would be a better choice.

Reply:
We can let user specify their desired initial values as block annotations, if such value is not set by user, we turn to use `0` instead.

Reply:
what kind of data types do we need to support currently?type info is needed when the user doesn't offer a default value,so we need to set the default value to 0 with a specific type according to the dtype of the sparse buffer.